snapshotstore/sql/sql.go

```diff
@@ -0,0 +1,106 @@
+package sql
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"reflect"
+
+	"github.com/hallgren/eventsourcing"
+	"github.com/hallgren/eventsourcing/snapshotstore"
+)
+
+type SQL struct {
```

Comment:
The exported types, functions and methods lacks documentation.
A small comment would be nice.